src/components/shared/OmniPicker/OmniPicker.tsx

```diff
@@ -129,55 +129,67 @@ class OmniPicker extends Component<Props> {
     }
   }
 
+  /**
+   * @NOTE All of the following methods are actually used by the wrapped component,
+   * eslint just doesn't seem to be able to pick it up
+   */
+  // eslint-disable-next-line react/no-unused-class-component-methods
```

Comment:
Could turn this off for the entire block instead of multiple next-lines?

Reply:
It can even be disabled for the whole file